Majasdarbs/03-majasdarbs/OlegsKozlovs/Portfolio/css/index.css

```diff
@@ -0,0 +1,2 @@
+@import "./normalize.css";
+@import "./style.css";
```

Comment:
without "./" , just @import "normalize.css" and @import "style.css". Because they are located in same folder